app/models/rubygem.rb

```diff
@@ -335,6 +337,7 @@ def reverse_runtime_dependencies
     reverse_dependencies.where("d.scope ='runtime'")
   end
 
+  # TODO: rename this to clarify owner-defined mfa requirement vs policy mandated mfa requirement
```

Comment:
I quickly grepped through the repo to see if there are any `TODO` comments. It looks like there's not many, just a handful. I'm hesitant about adding a TODO in the RubyGems codebase. If we're meaning to tackle this in the near future, I'd recommend opening an issue instead of adding this TODO inline with the code. Thoughts?